qa-tests-backend/src/test/groovy/ProcessBaselinesTest.groovy

```diff
@@ -228,12 +228,17 @@ class ProcessBaselinesTest extends BaseSpecification {
         List<ProcessBaselineOuterClass.ProcessBaseline> lockProcessBaselines = ProcessBaselineService.
                  lockProcessBaselines(clusterId, deployment, containerName, true)
         assert (!StringUtils.isEmpty(lockProcessBaselines.get(0).getElements(0).getElement().processName))
-        // sleep 5 seconds to allow for propagation to sensor
-        sleep 5000
+        // sleep 30 seconds to allow for propagation to sensor
+        sleep 30000
         orchestrator.execInContainer(deployment, "pwd")
 
+        log.info "Locked Process Baseline: ${lockProcessBaselines}"
+
+        // Give the reprocessing some time.
+        sleep 30000
```

Comment:
Do we need this? In next line we wait so we can increase timeout

Reply:
@janisz yeah I hear what you are saying and I get it.  I wrote ROX-11143 so that we can pass the nightly and work on it at the same time.  It takes forever to run those jobs and half the time the provision jobs fail.  So right or wrong, my theory was get it passing now and then work on some debug and tracing so I can optimize it in a more leisurely fashion.  Make sense?